.github/workflows/integration_tests.yml

```diff
@@ -112,13 +112,17 @@ jobs:
       contents: write
       packages: write
     environment: CI
-    runs-on: [ self-hosted, linux ]
+    runs-on: ubuntu-20.04
     needs: [ steward-build, hardhat-build ]
     strategy:
       fail-fast: false
       matrix:
         test_type: [ "AaveV2Stablecoin", "CellarV1" ]
     steps:
+      - name: Set up Go 1.16
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.16
```

Comment:
Nit: We saw earlier that at least one of our repos is 1.15. Not sure if we care if this is canonical in CI.